product_description_without_description_sale/models/product_template.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    use_description_sale = fields.Boolean(default=True)
```

Comment:
The default should be false.

Please add a system parameter to take the default value from (and add a demo record to update the parameter record) in case directly assigning false breaks tests.

```suggestion
    use_description_sale = fields.Boolean(
        "Use Sales Description",
        help="If selected, Sales Description will be appended to the sales order line description.",
    )
```